src/controls/auth/signup.ts

```diff
@@ -3,25 +3,39 @@ import prompt from "prompt-sync";
 import { Keys } from "../../commands/key";
 import { config } from "../../config/config";
 import localConfigService from "./svc";
+import PromptSyncPlus from "prompt-sync-plus";
+
+const c = {
+  echo: "* "
+} as Parameters<typeof PromptSyncPlus>[0];
 
 const signup = async () => {
   const email = prompt()("Enter your email: ");
-  const password = prompt().hide("Enter your password: ");
+  const password = prompt()({ ask: "Enter your password: ", echo: "* " });
   const userName = prompt()("Enter your username: ");
-  const apiKey = prompt().hide(
+  const openAIKey = PromptSyncPlus(c)(
```

Comment:
nit: I don't think its intuitive that a user is going to be asked for a anthropic key after being asked for an openAI key. In the future when we have more providers I think it makes sense to inform them which providers we support during the setup process.

Reply:
Yeah, i agree. I think in the future we just set them up with Forge key and allow them to pass in openAI key at runtime if they want to use their own.